### PR TITLE
Added over a dozen missing Berkeley faculty

### PR DESCRIPTION
**Pull request by ghost**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
A number of these faculty are on non-sabbatical leave (Dawn) and thus are below 75% time, or are not in CS (Barna), so they do not meet the inclusion criteria. Please update accordingly.

**Comment by ghost** [comment]
Thanks for getting back to me! I checked all over Berkeley's department website and also Dawn's personal webpage, and I couldn't find any mention of her being on leave. So I think maybe she's not on leave anymore? Also I included Barna Saha (in the IEOR department) and Nikhil Srivastava (in the math department) because they are both listed as part of Berkeley's theory group here-http://theory.cs.berkeley.edu/ and they are also able to advise CS PhD students (and they have in the past).

**Comment by emeryberger** [comment]
Dawn personally confirmed to me that she is on leave (or at least, was, and I asked her to let me know when she returned). Barna is not listed as a CS faculty member, so I would need to see some confirmation that she can _solely_ advise a PhD _in Computer Science_, and then she could be added.

**Comment by ghost** [comment]
I'll update those changes accordingly then. Sorry about that!

**Comment by ghost** [comment]
Done!

File: csrankings.csv [hunk 1]
```diff
@@ -20699,4 +20699,16 @@ m. c. schraefel,University of Southampton,https://www.ecs.soton.ac.uk/people/sig
...
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
-Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Jitendra Malik,University of California - Berkeley,https://people.eecs.berkeley.edu/~malik/,oY9R5YQAAAAJ
+Avishay Tal,University of California - Berkeley,http://www.avishaytal.org/,4pytJD8AAAAJ
+Krste Asanovic,University of California - Berkeley,http://people.eecs.berkeley.edu/~krste/,e4I7ihkAAAAJ
+Jacob Steinhardt,University of California - Berkeley,https://www.stat.berkeley.edu/~jsteinhardt/,LKv32bgAAAAJ
+Yakun Sophia Shao,University of California - Berkeley,https://people.eecs.berkeley.edu/~ysshao/,YoSGjoUAAAAJ
+Dan Klein,University of California - Berkeley,https://people.eecs.berkeley.edu/~klein/,jJRiZR8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:20708]: Already present in DB.

File: csrankings.csv [hunk 2]
```diff
@@ -20699,4 +20699,16 @@ m. c. schraefel,University of Southampton,https://www.ecs.soton.ac.uk/people/sig
 Öznur Tastan,Sabancı University,http://people.sabanciuniv.edu/otastan,8ljjXmEAAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
-Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Jitendra Malik,University of California - Berkeley,https://people.eecs.berkeley.edu/~malik/,oY9R5YQAAAAJ
+Avishay Tal,University of California - Berkeley,http://www.avishaytal.org/,4pytJD8AAAAJ
+Krste Asanovic,University of California - Berkeley,http://people.eecs.berkeley.edu/~krste/,e4I7ihkAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:20705]: RISC-V - not clear if full-time (see `old/industry.csv`).

File: csrankings.csv [hunk 3]
```diff
@@ -20699,4 +20699,16 @@ m. c. schraefel,University of Southampton,https://www.ecs.soton.ac.uk/people/sig
...
-Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Jitendra Malik,University of California - Berkeley,https://people.eecs.berkeley.edu/~malik/,oY9R5YQAAAAJ
+Avishay Tal,University of California - Berkeley,http://www.avishaytal.org/,4pytJD8AAAAJ
+Krste Asanovic,University of California - Berkeley,http://people.eecs.berkeley.edu/~krste/,e4I7ihkAAAAJ
+Jacob Steinhardt,University of California - Berkeley,https://www.stat.berkeley.edu/~jsteinhardt/,LKv32bgAAAAJ
+Yakun Sophia Shao,University of California - Berkeley,https://people.eecs.berkeley.edu/~ysshao/,YoSGjoUAAAAJ
+Dan Klein,University of California - Berkeley,https://people.eecs.berkeley.edu/~klein/,jJRiZR8AAAAJ
+John DeNero,University of California - Berkeley,http://denero.org/,eAwnN44AAAAJ
+Hany Farid,University of California - Berkeley,https://farid.berkeley.edu/,3OKn_UYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:20710]: Already in DB.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please check to see who is in the DB and remove everyone who is already present. Also, please place in alphabetical order to avoid conflicts.

**Comment by emeryberger** [comment]
Bump @zaidahmad23

**Comment by ghost** [comment]
Hey Emery, sorry for all the hassle and inconvenience. I've arranged everything in alphabetical order and gotten rid of the duplicates. I cleared out some files on my laptop and accidentally messed with some of my repositories on Github so I've ended up having to make a new pull request.

**Pull request closed without merging**